docs/source/scribe_data/cli.rst

```diff
@@ -54,16 +54,86 @@ Example output:
 .. code-block:: text
 
     $ scribe-data list
+
     Language     ISO  QID
     -----------------------
     English      en   Q1860
     French       fr   Q150
     German       de   Q188
-    ...
+    Italian      it   Q652
```

Comment:
I'll change these to just English with ... in the next row as soon the full output will be way to long.

Reply:
Okay